packages/coordinator/src/screens/AuthScreens/ResetPasswordScreen.tsx

```diff
@@ -62,13 +70,68 @@ export default function ResetPasswordScreen(props: ResetPasswordScreenProps) {
           <div>
             <Button
               title={"איפוס סיסמה"}
-              onClick={resetPassword}
+              onClick={handleResetPass}
               isLoading={isLoading}
               isDisabled={!email}
             />
           </div>
         </div>
       </div>
+      <Popup
+        open={successPopupOpen}
+        onClose={() => setSuccessPopupOpen(false)}
+        content={
+          <div className={styles.resetPassPopupContent}>
+            <img
+              src={emailOnTheWay}
+              alt=""
+              className={styles.resetPassPopupImg}
+            />
+            <div className={styles.resetPassPopupTitle}>
+              מייל איפוס בדרך אליך
+            </div>
+            <div className={styles.resetPassPopupMessage}>
+              ברגעים הקרובים יישלח אליך מייל עם לינק לאיפוס הסיסמא
+            </div>
+            <Button
+              className={styles.resetPasswordScreenPopupButton}
+              onClick={() => {
+                setEmailError("");
+                setSuccessPopupOpen(false);
+                handleResetPass();
+              }}
+              title="לא קיבלתי, שלחו לי שוב"
+              variant={ButtonVariant.text}
+            />
+          </div>
+        }
+      />
+      <Popup
+        open={errorPopupOpen}
+        onClose={() => setErrorPopupOpen(false)}
+        content={
+          <div className={styles.resetPassPopupContent}>
+            <img
+              src={emailNotFound}
+              alt=""
+              className={styles.resetPassPopupImg}
+            />
+            <div className={styles.resetPassPopupTitle}>אופס</div>
+            <div className={styles.resetPassPopupMessage}>
+              כתובת המייל שלך לא נמצאה
```

Comment:
Is this the only possible error?
Don't you want to print here `emailError`?

Reply:
done